packages/blade/src/components/Badge/Badge.tsx

```diff
@@ -19,7 +22,7 @@ type BadgeProps = {
    *
    * @default 'neutral'
    */
-  variant?: 'positive' | 'negative' | 'notice' | 'information' | 'neutral';
+  variant?: Feedback | 'blue';
```

Comment:
Confused: We also call these intent at some places, should we use that terminology instead of `Feedback`? If not when should we call these intent and when feedback?